Dockerfile-dev

```diff
@@ -0,0 +1,26 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+
+RUN apt-get -y update
+RUN apt-get install -y --fix-missing build-essential cmake libpq-dev curl
+
+RUN pip install 'poetry==1.1.7'
+
+ENV MODULE_NAME="jobbergateapi2.main"
+
+COPY ./pyproject.toml /app/
+COPY ./poetry.lock /app/
+
+ARG PYPI_PASSWORD
+ENV POETRY_HTTP_BASIC_PYPICLOUD_USERNAME=admin
+ENV POETRY_HTTP_BASIC_PYPICLOUD_PASSWORD=$PYPI_PASSWORD
+
+ARG AWS_ACCESS_KEY_ID
+ENV AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+
+ARG AWS_SECRET_ACCESS_KEY
+ENV AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
```

Comment:
should these be added by `hack_build_secrets.sh` too? Or is it just easier in local dev to pull the stuff in from args?

Reply:
You are absolutely right

Reply:
Well, in the prod container, it should be pulling AWS stuff from docker secrets. Unlike the PYPI_PASSWORD, these aren't needed at container build time.

For local build it seemed easy at the moment to just include them as a build-arg.